src/app/login-signup/routes/routes.ts

```diff
@@ -0,0 +1,7 @@
+import { Routes } from '@angular/router';
+
+import { MainComponent } from '../main/main.component';
+
+export const LoginRoutes: Routes  = [
+    {path: 'login', component: MainComponent}
```

Comment:
For consistency...
```suggestion
    { path: 'login', component: MainComponent }
```
https://github.com/Vehmloewff/jikno/blob/master/src/app/routing/app-routing.module.ts#L8